### PR TITLE
Deflatten repository structure

### PR DESCRIPTION
**Pull request by ADSteele916**
- Rearranged repository structure to have fewer files at the root. This makes things cleaner and more explicitly conveys the usages of the various modules.
- Don't merge until #20 is merged. If you start any new branches from now on, branch from deflattening.

### CODE REVIEW

**Comment by ADSteele916** [comment]
Sorted out the merge conflicts with master. Everything should be ready to merge whenever we're ready.

File: main_window/rocket_data.py [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
-from detail import *
-from SubpacketIDs import SubpacketEnum
+from detail import LOCAL
+
+from . import subpacket_ids
+from .subpacket_ids import SubpacketEnum
+
+
+if not os.path.exists(os.path.join(LOCAL, "logs")):
+    os.mkdir(os.path.join(LOCAL, "logs"))
```
[the remark this answers is not in the document]
Comment (ADSteele916) [reply main_window/rocket_data.py]: Done.

File: main_window/main.py [hunk 2]
```diff
@@ -268,3 +263,13 @@ def mapResizedEvent(self, event) -> None:
         :type event:
         """
         self.MappingThread.setDesiredMapSize(event.width, event.height)
+
+
+window = None
+
+
+def start(*args, **kwargs):  # TODO: does this need its own file??
```
Comment (SilentSys) [inline main_window/main.py:271]:
```suggestion
def start(*args, **kwargs):
```

This comment was only relevant when it was in the start.py file. It was just questioning whether it was even necessary. But now I think the start file actually has a good purpose.

**Review by SilentSys — CHANGES_REQUESTED** [review]
LGTM! Thanks for combing through all this!

**Review by SilentSys — APPROVED** [review]

File: main_window/main.py (hunk 2 above)
Comment (ADSteele916) [reply main_window/main.py]: Good catch! We certainly don't want any more TODOs lying around.

**Pull request merged**